core/config/env/env.go

```diff
@@ -20,6 +20,7 @@ var (
 	ThresholdKeyShare            = Secret("CL_THRESHOLD_KEY_SHARE")
 	// Migrations env vars
 	EVMChainIDNotNullMigration0195 = "CL_EVM_CHAINID_NOT_NULL_MIGRATION_0195"
+	CustomDefaults                 = Var("CL_CHAIN_DEFAULTS")
```

Comment:
Should this include `EVM`? Or is it set up to be generalized for other chains later?

Reply:
> Or is it set up to be generalized for other chains later?

👍

Reply:
How so? I would think we'd need to be forward thinking about expecting an `evm/` directory or something like that.

Reply:
> I would think we'd need to be forward thinking about expecting an evm/ directory or something like that.

Yea that's what I'm imagining, but we can look for the presence of an evm directory in code w/o bloating the number of configurable env vars.

Reply:
Let's start by expecting an `evm/` dir now so that in the future we don't have to move things or carry around extra exceptional code.